.doctrine-project.json

```diff
@@ -5,21 +5,39 @@
     "docsSlug": "doctrine-coding-standard",
     "versions": [
         {
-            "name": "9.0",
-            "branchName": "9.0.x",
-            "slug": "9.0",
+            "name": "11.0",
+            "branchName": "11.0.x",
+            "slug": "11.0",
             "upcoming": true
         },
         {
-            "name": "8.2",
-            "branchName": "8.2.x",
-            "slug": "8.2",
+            "name": "10.1",
```

Comment:
I'm not sure this one will be worth it, or how we should proceed in that kind of case. Maybe using the patch branch as the default for this repository is not the best idea either.

Reply:
Do we have a need for a 10.1 branch now?

Reply:
Nobody asked for it.